src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

Create interfaces for the behavior of getArea() and draw()